### PR TITLE
[Docs] Trivia List Creation User Guide

### LINKED ISSUE #4595: [Trivia] Trivia list creation guide
Opening this to track work on creating a trivia list creation guide, which given other pending issues relating to trivia (namely #4571 and #4580 which if accepted and those instructions had already been written would've required additions to/a rewrite of the instructions), it seemed to not be a good idea to include in #4566 at this time

### PR DESCRIPTION
**Pull request by Kreusada**
A guide for creating trivia lists, closes #4595.

Adaptable for changes via #4580.

### CODE REVIEW

File: docs/guide_trivia_list_creation.rst [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+.. image:: .resources/trivia/trivia_author.png
+
+The following should be placed at the top of your file, replacing "Red" 
+with your name:
+
+.. code-block:: yaml
+
+    AUTHOR: Red
+
+If there are multiple authors, we can seperate them with commas.
```
Comment (Flame442) [inline docs/guide_trivia_list_creation.rst]:
```suggestion
If there are multiple authors, we can separate them with commas.
```

File: docs/guide_trivia_list_creation.rst [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+Questions should consist of at least one answer, with other
+possible answers included if necessary. You must put a colon at the end 
+of the question, for example:
+
+.. code-block:: yaml
+
+    How many days are there in a regular year?:
+
+Answers will follow below, each seperated by a line break and with a
```
Comment (Flame442) [inline docs/guide_trivia_list_creation.rst]:
```suggestion
Answers will follow below, each separated by a line break and with a
```

File: docs/guide_trivia_list_creation.rst [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+hyphen at the start of the line.
+
+.. code-block:: yaml
+
+    How many days are there in a regular year?:
+    - 365
+    - three hundred and sixty five
+
+Its always nice to include alternative answers if a question needs it.
```
Comment (Flame442) [inline docs/guide_trivia_list_creation.rst]:
```suggestion
It's always nice to include alternative answers if a question needs it. 
```

File: docs/guide_trivia_list_creation.rst [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+.. code-block:: yaml
+
+    How many days are there in a regular year?:
+    - 365
+    - three hundred and sixty five
+
+Its always nice to include alternative answers if a question needs it. 
+We can add as many valid answers as we'd like below this question. Answers
+are **NOT** case sensitive, so you don't need to worry about adding the same
+answer multiple times in different casing.
```
Comment (Flame442) [inline docs/guide_trivia_list_creation.rst]:
```suggestion
answer multiple times in different casings.
```

**Review by Flame442 — CHANGES_REQUESTED** [review]
A few typos, but otherwise ok.

**Comment by Kreusada** [comment]
Thanks for reviewing, I've made those changes.

**Review by Flame442 — APPROVED** [review]
All set, thanks!

**Pull request merged**